lib/virtualbadge/view/animated_badge.dart

```diff
@@ -4,7 +4,10 @@ import 'package:flutter/material.dart';
 import 'package:provider/provider.dart';
 
 class AnimationBadge extends StatefulWidget {
-  const AnimationBadge({super.key});
+  final String text;
```

Comment:
Animation badge should have no relation to the text. It just displays whatever binary data is passed on